examples/D3d_tanana.ipynb

```diff
@@ -0,0 +1,1524 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b0ab8c8b-dd15-4321-920c-ce473aa3b5d1",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\eabrow\\AppData\\Roaming\\Python\\Python38\\site-packages\\dolfyn\\velocity.py:10: AccessorRegistrationWarning: registration of accessor <class 'dolfyn.velocity.Velocity'> under name 'velds' for type <class 'xarray.core.dataset.Dataset'> is overriding a preexisting attribute with the same name.\n",
+      "  class Velocity():\n"
+     ]
+    }
+   ],
+   "source": [
+    "from mhkit import dolfyn as dlfn\n",
+    "from dolfyn.adp import api\n",
```

Comment:
This is probably the error. "mhkit.dolfyn.adp"

Reply:
Thanks! I got it to work.

Reply:
Thank you @jmcvey3